### ISSUE TITLE
Integrate With Open Collective when possible.

### ISSUE THREAD

**Issue by karan-ta** [simonmichael/hledger#542]
Open Collective looks awesome and could be the future of companies.

It encourages being transparent about your finances.

Hledger should track open collective api release.
Once opencollective has an api - hledger should integrate with the opencollective api.

**Comment by simonmichael** [comment]
Agreed, that will be a great feature. An addon command might be the easiest place to prototype it. How about experimenting with some other APIs that exist already (paypal ?)

**Comment by simonmichael** [comment]
(See also http://hackage.haskell.org/package/paypal-rest-client or http://hackage.haskell.org/package/paypal-api)

**Comment by nobodyinperson** [comment]
This issue is done, isn't it?

**Comment by simonmichael** [comment]
We use Open Collective to collect funds for the hledger project, but I think this issue is about developing an integration making it easy for hledger users to import data from Open Collective. 

They do seem to have an API now, https://docs.opencollective.com/help/contributing/development/api, so this would be a matter of someone building a client, or more likely finding an existing one and writing up a how-to for using it with hledger.

Related: here are [CSV rules](https://github.com/simonmichael/hledger_finance/blob/main/oc.csv.rules) and the [manual download process](https://github.com/simonmichael/hledger_finance/blob/main/Makefile#L15) and makefile I currently use.
